### ISSUE TITLE
Test definitions - linting results

### ISSUE THREAD

**Issue by rartych** [camaraproject/NumberVerification#143]
**Problem description**
While testing linting rules for Gherkin the following output was produced for _NumberVerification_verify.feature_

```
  2      Multiple empty lines are not allowed                                                    no-multiple-empty-lines
  19     Multiple empty lines are not allowed                                                    no-multiple-empty-lines
  20     Multiple empty lines are not allowed                                                    no-multiple-empty-lines
  42     Trailing spaces are not allowed                                                         no-trailing-spaces
  52     Multiple empty lines are not allowed                                                    no-multiple-empty-lines
  81     Multiple empty lines are not allowed                                                    no-multiple-empty-lines
  160    Scenario name is already used in: NumberVerification_verify.feature:111                 no-dupe-scenario-names
  191    Trailing spaces are not allowed                                                         no-trailing-spaces
  192    Multiple empty lines are not allowed                                                    no-multiple-empty-lines
```
For _NumberVerification_device_phone_number_share.feature_
`  2    Multiple empty lines are not allowed                                                                       no-multiple-empty-lines`

**Expected behavior**
The most of issues are related to formatting.
But the optional scenario title is indeed duplicated in lines 111 and 160.

It should be considered to change the filenames of test definitions to kebab-case.

**Alternative solution**

**Additional context**

**Comment by bigludo7** [comment]
@rartych I've fixed. The check on the feature file is also reflected now on the check report?

**Comment by rartych** [comment]
Closing as fixed within release PR #139
